src/test/erasure-code/test-erasure-code.sh

```diff
@@ -30,16 +30,10 @@ function run() {
 
     setup $dir || return 1
     run_mon $dir a || return 1
-    # check that erasure code plugins are preloaded
-    CEPH_ARGS='' ceph --admin-daemon $dir/ceph-mon.a.asok log flush || return 1
-    grep 'load: jerasure.*lrc' $dir/mon.a.log || return 1
```

Comment:
Yes, I think, this issue can take place, thanks for link. I'll add preload code in this case (it doesn't exist in new PluginRegistry class).